docs/design/low-overhead-deployment-for-server-mode/low-overhead-deployment-for-server-mode.md

```
@@ -25,7 +33,25 @@
+    * [U8](#u8)
+    * [U9](#u9)
+    * [U10](#u10)
+    * [U11](#u11)
+    * [U12](#u12)
+* [Available solutions](#available-solutions)
+    * [S1](#s1)
+    * [S2](#s2)
+    * [S3](#s3)
+    * [S4](#s4)
+    * [S5](#s5)
+* [Chosen approach and configuration](#chosen-approach-and-configuration)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)